tracex/extraction/prototype/create_all_trace_xes.py

```diff
@@ -32,15 +32,16 @@ def create_all_trace_xes(csv_file, key):
             "duration": "time:duration",
         }
     )
-    output_name = "all_traces_" + key + ".xes"
+    output_name = "all_traces_" + key + suffix + ".xes"
     pm4py.write_xes(
         dataframe,
         (u.out_path / output_name),
         case_id_key="case:concept:name",
         activity_key="concept:name",
         timestamp_key="time:timestamp",
     )
+    return str(u.out_path / output_name)
 
 
-ACTIVITY_KEY = get_key()
-create_all_trace_xes(u.CSV_OUTPUT, ACTIVITY_KEY)
+# ACTIVITY_KEY = 'event_type' # get_key()
+# create_all_trace_xes(u.CSV_OUTPUT, ACTIVITY_KEY)
```

Comment:
This needs to be resolved. Theses lines where a problem in the UI, but seem to necessary in the command line tool. We need to find a way to make the execution of this function not mandatory.
Maybe the call can be made explicitly in main?

Reply:
As we do not work with xes in the terminal variant of the pipeline, we never call get_key anyway. I would remove two lines, but keep the function as it could come in handy when creating different dfgs later on

Reply:
Very well, we might also use this, once we provide the option to select the activity key in the UI.